accelerator/bxb/board/ultra96/Makefile

```diff
@@ -0,0 +1,30 @@
+CHISEL_FILES := $(wildcard ../../src/main/scala/**/*.scala)
+
+
+.NOTPARALLEL: all
+
+all:  rtl/Bxb.v vivado vivado_gen
+
+rtl/Bxb.v: $(CHISEL_FILES)
+	cd ../../ && sbt "runMain bxb.Bxb --output-file $(abspath $@)"
+
+.PHONY: vivado
+vivado: 
+	vivado -mode tcl -source bxb.tcl
+#	rm -rf project_1/
+#	rm -rf vivado*
+
+.PHONY: vivado_gen
+vivado_gen:
```

Comment:
Does this one depend on `vivado` run?
Could we add dependency between in and this target somehow?

Reply:
Update Makefile.